### PR TITLE
[Docathon] Fix BCEWithLogitsLoss API Docs

### PR DESCRIPTION
**Pull request by Mister-TT**
PR types
Bug fixes

PR changes
Docs

Description：
1 与英文对照，使翻译更通顺 
2 修正标点 
3 注意 处使用.. note::

Issue from：「文档评测」NO.1

涉及到的文档有：
docs\api\paddle\nn\BCEWithLogitsLoss_cn.rst

相关链接：
https://github.com/PaddlePaddle/docs/issues/6187
https://github.com/PaddlePaddle/docs/issues/6165

@sunzhongkai588

### CODE REVIEW

File: docs/api/paddle/nn/BCEWithLogitsLoss_cn.rst [hunk 1]
```diff
@@ -7,16 +7,16 @@ BCEWithLogitsLoss
 
 可用于创建一个 BCEWithLogitsLoss 的可调用类，计算输入 `logit` 和标签 `label` 间的 `binary cross entropy with logits loss` 损失。
```
Comment (ooooo-create) [inline docs/api/paddle/nn/BCEWithLogitsLoss_cn.rst]: > 计算输入 `logit` 和标签 `label` 间的 `binary cross entropy with logits loss` 损失。

这部分直接显示 logit ，用户都不知道是不是专有名称，后面也没有解释，感觉 `计算输入的预测值 logit` 比较好

**Review by ooooo-create — COMMENTED** [review]
nice work~
下面的
 >   - **logit** (Tensor) - :math:`[N, *]`，其中 N 是 batch_size， `*` 是任意其他维度

也可以添加一下 logit 的描述~

File: docs/api/paddle/nn/BCEWithLogitsLoss_cn.rst [hunk 2]
```diff
@@ -7,16 +7,16 @@ BCEWithLogitsLoss
...
 
-结合了 `sigmoid` 操作和 :ref:`api_nn_loss_BCELoss` 操作。同时，我们也可以认为这是 ``sigmoid_cross_entrop_with_logits`` 和一些 `reduce` 操作的组合。
+计算方式结合了 `sigmoid` 操作和 :ref:`api_nn_loss_BCELoss` 操作。或者，我们也可以认为计算方式是 ``sigmoid_cross_entrop_with_logits`` 和一些 `reduce` 操作的组合。
```
Comment (ooooo-create) [inline docs/api/paddle/nn/BCEWithLogitsLoss_cn.rst]:
```suggestion
计算方式结合了 `sigmoid` 操作和 :ref:`cn_api_paddle_nn_BCELoss` 操作。或者，我们也可以认为计算方式是 ``sigmoid_cross_entrop_with_logits`` 和一些 `reduce` 操作的组合。
```
修复一个 api_label 的错误引用。link #6170 和 #6142

Comment (Mister-TT) [reply docs/api/paddle/nn/BCEWithLogitsLoss_cn.rst]: 好的好的，我之前不知道可以commit suggestion来采用该修改。我现在是直接在新的merge里面按照这个方式改了，劳烦审查啦 ~

File: docs/api/paddle/nn/BCEWithLogitsLoss_cn.rst [hunk 3]
```diff
@@ -5,18 +5,18 @@ BCEWithLogitsLoss
...
 .. py:class:: paddle.nn.BCEWithLogitsLoss(weight=None, reduction='mean', pos_weight=None, name=None)
 
-可用于创建一个 BCEWithLogitsLoss 的可调用类，计算输入 `logit` 和标签 `label` 间的 `binary cross entropy with logits loss` 损失。
+可用于创建一个 BCEWithLogitsLoss 的可调用类，计算输入的预测值 `logit` 和标签 `label` 间的 `binary cross entropy with logits loss` 损失。
 
-结合了 `sigmoid` 操作和 :ref:`api_nn_loss_BCELoss` 操作。同时，我们也可以认为这是 ``sigmoid_cross_entrop_with_logits`` 和一些 `reduce` 操作的组合。
+计算方式结合了 `sigmoid` 操作和 :ref:`cn_api_paddle_nn_BCELoss` 操作。或者，我们也可以认为计算方式是 ``sigmoid_cross_entrop_with_logits`` 和一些 `reduce` 操作的组合。
 
-在每个类别独立的分类任务中，可以用于计算按元素的概率误差。可以将其视为预测数据点的标签，其中标签不是互斥的。例如，一篇新闻文章可以同时关于政治，科技，体育或者同时不包含这些内容。
+该损失函数衡量了在每个类别相互独立的分类任务中的逐元素概率误差。这可以被视为对数据点进行标签预测，其中标签不是相互排斥的。例如，一篇新闻文章可以同时关于政治，科技，体育或者同时不包含这些内容。
```
Comment (sunzhongkai588) [inline docs/api/paddle/nn/BCEWithLogitsLoss_cn.rst]:
```suggestion
该损失函数衡量了在每个类别相互独立的分类任务中的逐元素概率误差。这可以被视为预测数据点的标签，其中标签不是相互排斥的。例如，一篇新闻文章可以同时关于政治，科技，体育或者同时不包含这些内容。
```

File: docs/api/paddle/nn/BCEWithLogitsLoss_cn.rst [hunk 4]
```diff
@@ -30,20 +30,21 @@ BCEWithLogitsLoss
...
     - **name** (str，可选) - 具体用法请参见 :ref:`api_guide_Name`，一般无需设置，默认值为 None。
 
 形状
 :::::::::
-    - **logit** (Tensor) - :math:`[N, *]`，其中 N 是 batch_size， `*` 是任意其他维度。输入数据 ``logit`` 一般是线性层的输出，不需要经过 ``sigmoid`` 层。数据类型是 float32、float64。
-    - **label** (Tensor) - :math:`[N, *]`，标签 ``label`` 的维度、数据类型与输入 ``logit`` 相同。
-    - **output** (Tensor) - 输出的 Tensor。如果 :attr:`reduction` 是 ``'none'``，则输出的维度为 :math:`[N, *]`，与输入 ``input`` 的形状相同。如果 :attr:`reduction` 是 ``'mean'`` 或 ``'sum'``，则输出的维度为 :math:`[]` 。
+    - **logit** (Tensor) - 预测值，维度是 :math:`[N, *]`，其中 N 是 batch_size， `*` 是任意其他维度。预测值 ``logit`` 一般是线性层的输出，不需要经过 ``sigmoid`` 层。数据类型是 float32、float64。
+    - **label** (Tensor) - 标签， 维度是 :math:`[N, *]`，标签 ``label`` 的维度，数据类型与预测值 ``logit`` 相同。
+    - **output** (Tensor) - 输出。 如果 :attr:`reduction` 是 ``'none'``，则输出的维度为 :math:`[N, *]`，与 ``logit`` 的形状相同。如果 :attr:`reduction` 是 ``'mean'`` 或 ``'sum'``，则输出的维度为 :math:`[]` 。
```
Comment (sunzhongkai588) [inline docs/api/paddle/nn/BCEWithLogitsLoss_cn.rst]:
```suggestion
    - **logit** (Tensor) - 输入的预测值。形状是 :math:`[N, *]`，其中 N 是 batch_size， `*` 是任意其他维度。输入的预测值 ``logit`` 一般是线性层的输出，不需要经过 ``sigmoid`` 层。数据类型是 float32、float64。
    - **label** (Tensor) - 标签。 形状是 :math:`[N, *]`，标签 ``label`` 的维度、数据类型与输入的预测值 ``logit`` 相同。
    - **output** (Tensor) - 输出的 Tensor。 如果 :attr:`reduction` 是 ``'none'``，则输出的维度为 :math:`[N, *]`，与输入 ``logit`` 的形状相同。如果 :attr:`reduction` 是 ``'mean'`` 或 ``'sum'``，则输出的维度为 :math:`[]` 。
```

**Review by sunzhongkai588 — APPROVED** [review]
LGTM
